pnrm/plugins/swfwlicense/licensemgr/pano.go

```diff
@@ -0,0 +1,137 @@
+package licensemgr
+
+import (
+	"encoding/xml"
+	"github.com/PaloAltoNetworks/pango/namespace"
+	"github.com/PaloAltoNetworks/pango/util"
+)
+
+// Panorama is the client.Panorama namespace.
+type Panorama struct {
+	ns *namespace.Plugin
+}
+
+// GetBootstrapParams for specific licensemgr and returns a string of the bootstrap parameters and an error if needed
+func (c *Panorama) GetBootstrapParams(licensemgr string) (string, error) {
```

Comment:
Rename this param to `manager` or something; this variable is named the same as this whole package.

Reply:
I've left this as `licensemgr` as renamed the package to `manager`, so it differentiates between the two now.